backend/models/bookModel.js

```diff
@@ -0,0 +1,48 @@
+const mongoose = require('mongoose')
+
+const Schema = mongoose.Schema;
+
+const bookSchema = new Schema({
+    title: {
+        type: String,
+        required: true,
+        unique: true,
+        trim: true
+    },
+    author:{
+        type: String,
+        required: true,
+    },
+    description: {
+        type: String,
+        required: true,
+    }, 
+    bookImg:{
+        type: String,
+        required: true,
+    },
+    genre: {
+        type:String
+    },
+    tags:{
+        type: String,
```

Comment:
is this supposed to be a list of tags?

Reply:
yes lol is that bad?

Reply:
i'm not that familiar with mongoose, but it doesn't seem like it's a list